### PR TITLE
add more tests

### PR DESCRIPTION
**Pull request by derwehr**
Adds tests for the EntityDetailsStore, still WIP

### CODE REVIEW

File: frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      expect(actual).to.be.eq(expected);
+    });
+    it('should return details of the node with id 1', () => {
+      // Arrange
+      entityDetails = container.get(EntityDetails);
+      cy.intercept('http://localhost:8080/api/getNodesById*', id1);
+      entityDetailsState.setState({ node: 1, edge: null });
+
+      // Act
+      const actual = entityDetails.getValue();
```
Comment (derwehr) [inline frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts:33]: Does anyone know why `actual` is `null` here, after updating the StateStore in line 30?

Comment (joluj) [reply frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts:33]: I think the intercept is not working properly

Comment (derwehr) [reply frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts:33]: Think you're right. `intercept` doesn't seem to intercept requests from within the container. But I need the container for DI :(

Comment (joluj) [reply frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts:33]: You could overwrite the service with a mock service. There, return a static value, something like that:

```ts
class MockXyzService extends XyzService {
    getNodesById(...) { // This method is overwritten
        return of(mockValue);
    }
}
```

Comment (derwehr) [reply frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts:35]: Thank you! Got it

**Pull request merged**